content/blog/2018/app-metrics-istio/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Application Metrics in Istio
+description: How to send application metrics to Prometheus instance in Istio.
+publishdate: 2018-12-19
+subtitle:
+attribution: Mete Atamel
+twitter: meteatamel
+weight: 78
+keywords: [metrics, prometheus, mixer, envoy, sidecar, traffic management]
```

Comment:
Please remove all spaces in between the [], and put a dash in traffic-management.

Thanks.

Reply:
Ok, fixed.